docs/guide/data-security.asciidoc

```diff
@@ -0,0 +1,408 @@
+
+* Go: {apm-go-ref-v}/configuration.html#config-sanitize-field-names[`ELASTIC_APM_SANITIZE_FIELD_NAMES`]
+* Java: {apm-java-ref-v}/config-core.html#config-sanitize-field-names[`sanitize_field_names`]
+* .NET: {apm-dotnet-ref-v}/config-core.html#config-sanitize-field-names[`sanitizeFieldNames`]
+* Node.js: {apm-node-ref-v}/configuration.html#sanitize-field-names[`sanitizeFieldNames`]
+// * PHP: {apm-php-ref-v}[``]
+* Python: {apm-py-ref-v}/configuration.html#config-sanitize-field-names[`sanitize_field_names`]
+* Ruby: {apm-ruby-ref-v}/configuration.html#config-sanitize-field-names[`sanitize_field_names`]
+
+Alternatively, you can completely disable the capturing of HTTP headers.
+This setting also supports {kibana-ref}/agent-configuration.html[Central configuration]:
+
+* Go: {apm-go-ref-v}/configuration.html#config-capture-headers[`ELASTIC_APM_CAPTURE_HEADERS`]
+* Java: {apm-java-ref-v}/config-core.html#config-sanitize-field-names[`capture_headers`]
+* .NET: {apm-dotnet-ref-v}/config-http.html#config-capture-headers[`CaptureHeaders`]
+* Node.js: {apm-node-ref-v}/configuration.html#capture-headers[`captureHeaders`]
+// * PHP: {apm-php-ref-v}[``]
+* Python: {apm-py-ref-v}/configuration.html#config-capture-headers[`capture_headers`]
+* Ruby: {apm-ruby-ref-v}/configuration.html#config-capture-headers[`capture_headers`]
+
+[discrete]
+[[filter-http-body]]
+==== HTTP bodies
+
+By default, the body of HTTP requests is not recorded.
+Request bodies often contain sensitive data like passwords or credit card numbers,
+so use care when enabling this feature.
+
+This setting supports {kibana-ref}/agent-configuration.html[Central configuration],
+which means the list of sanitized fields can be updated without needing to redeploy your services:
+
+* Go: {apm-go-ref-v}/configuration.html#config-capture-body[`ELASTIC_APM_CAPTURE_BODY`]
+* Java: {apm-java-ref-v}/config-core.html#config-sanitize-field-names[`capture_body`]
+* .NET: {apm-dotnet-ref-v}/config-http.html#config-capture-body[`CaptureBody`]
+* Node.js: {apm-node-ref-v}//configuration.html#capture-body[`captureBody`]
+// * PHP: {apm-php-ref-v}[``]
+* Python: {apm-py-ref-v}/configuration.html#config-capture-body[`capture_body`]
+* Ruby: {apm-ruby-ref-v}/configuration.html#config-capture-body[`capture_body`]
+
+[discrete]
+[[filter-real-user-data]]
+==== Real user monitoring data
+
+Protecting user data is important.
+For that reason, individual RUM instrumentations can be disabled in the RUM agent with the
+{apm-rum-ref-v}/configuration.html#disable-instrumentations[`disableInstrumentations`] configuration variable.
+Disabled instrumentations produce no spans or transactions.
+
+[options="header"]
+|====
+|Disable |Configuration value
+|HTTP requests |`fetch` and `xmlhttprequest
+|Page load metrics including static resources |`page-load`
+|User browser errors |`error`
+|User click events including URLs visited, mouse clicks, and navigation events |`eventtarget`
+|====
```

Comment:
This could use a review from @vigneshshanmugam. Does this table look correct? Is there more I could add?

Reply:
Added some comments Brandon, looks good otherwise. 

Some more info can be found here - https://github.com/elastic/dev/issues/1443#issuecomment-657672433

Added this list for both RUM and Node.js long time ago.

Reply:
Fantastic, thank you!